frontend/device/android/device.lua

```diff
@@ -131,9 +131,17 @@ local Device = Generic:new{
 function Device:init()
     self.screen = require("ffi/framebuffer_android"):new{device = self, debug = logger.dbg}
     self.powerd = require("device/android/powerd"):new{device = self}
+
+    local event_map = require("device/android/event_map")
+
+    if android.prop.model == "tolino" then
+        android_eventmap[21] = "LPgBack" -- changed for Tolino Buttons (up key)
+        android_eventmap[22] = "LPgFwd" -- changed for Tolino Buttons (down key)
+    end
+
     self.input = require("device/input"):new{
         device = self,
-        event_map = require("device/android/event_map"),
+        event_map = event_map,
```

Comment:
and here?

Reply:
fixed with 42135c8b3a460aa4be80877ab0b3b61044b61a9c